RadarSDK/RadarTripOptions.m

```diff
@@ -89,7 +89,7 @@ - (NSDictionary *)dictionaryValue {
     dict[kDestinationGeofenceTag] = self.destinationGeofenceTag;
     dict[kDestinationGeofenceExternalId] = self.destinationGeofenceExternalId;
     dict[kMode] = [Radar stringForMode:self.mode];
-    dict[kScheduledArrivalAt] = self.scheduledArrivalAt;
```

Comment:
NSDate object does not get converted, this field was null, format it to string since all our other date formats are converted to string with JSON